### PR TITLE
add DE translations

### PR DESCRIPTION
**Pull request by chappjc**
This is @karamble's https://github.com/decred/dcrdex/pull/1815 but on master

The last commit is TODOs for a few updates that weren't included in his PR to `release-v0.5`

### CODE REVIEW

File: client/webserver/locales/de-de.go [hunk 1]
```diff
@@ -190,6 +190,7 @@ var DeDE = map[string]string{
 	"Your Deposit Address":        "Einzahlungsadresse für Ihr Wallet",
 	"Send enough for reg fee":     `Achte darauf genug senden um auch die Netzwerkgebühren zu decken. Du kannst so viel einzahlen wie du willst da im nächsten Schritt nur die Registrierungsgebühr abgezogen wird. Du musst die Bestätigungen deiner Einzahlung abwarten damit du fortfahren kannst.`,
 	"Send enough with estimate":   `Zahle mindestens <span data-tmpl="totalFees"></span> <span class="unit">XYZ</span> ein um auch die Netzwerkgebühren der Transaktion zu decken. Du kannst so viel einzahlen wie du möchtest da nur der erforderliche Betrag im nächsten Schritt verrechnet wird. Du musst die Bestätigungen deiner Einzahlung abwarten damit du fortfahren kannst.`,
+	"Send funds for token":        `Deposit at least <span data-tmpl="tokenFees"></span> <span class="unit">XYZ</span> and <span data-tmpl="parentFees"></span> <span data-tmpl="parentUnit">XYZ</span> to also cover fees. You may deposit as much as you like to your wallet, since only the required amount will be used in the next step. The deposit must confirm to proceed.`, // karamble
```
Comment (chappjc) [inline client/webserver/locales/de-de.go:193]: @karamble we misses this one.

File: client/webserver/locales/de-de.go [hunk 2]
```diff
@@ -244,4 +245,20 @@ var DeDE = map[string]string{
 	"export_wallet_msg":           "Nachfolgend findest du die erforderlichen Seeds um dein Wallet in einigen der beliebten externen Wallets wiederherzustellen. Führe KEINE Transaktionen mit deinen externen Wallet durch während du aktive Trades auf dem DEX laufen hast.",
 	"clipboard_warning":           "Das Kopieren/Einfügen eines Wallet-Seeds stellt ein potenzielles Sicherheitsrisiko dar. Dies geschieht auf eigene Gefahr.",
 	"fiat_exchange_rate_sources":  "Quellen für den Fiat-Wechselkurs",
+	"wallet_wait_synced":          "Das Wallet wird nach der Synchronisierung erstellt",
```
Comment (chappjc) [inline client/webserver/locales/de-de.go:248]: @karamble Sorry, I forgot one entirely.

https://github.com/decred/dcrdex/blob/b46222130c4956cfd1f2da1e07301cfcbd64671f/client/webserver/locales/en-us.go#L248

`"Synchronizing"`

**Comment by chappjc** [comment]
Just missing one more in client/webserver/locales/de-de.go:

```
$  go generate | grep de-DE
Prepping de-DE
Prepping de-DE
Prepping de-DE
Warning: no de-DE replacement text for key "Synchronizing", using 'en' value Synchronizing 
Prepping de-DE
Prepping de-DE
Prepping de-DE
Prepping de-DE
Prepping de-DE
Prepping de-DE
Prepping de-DE
```

**Comment by xaur** [comment]
I was mapping all translations for the [status page](https://decredcommunity.github.io/translations/status) and noticed that the DE translation PR was first submitted to the 0.5 release branch and then ported to `master` in this PR. Previously it was first merged to `master` and then ported to the release branch (this is also how Decrediton does it).

So it may be worth to mention in [Localization and Translation](https://github.com/decred/dcrdex/blob/master/docs/wiki/Localization-and-Translation.md) which branch translators should submit against.

File: client/webserver/locales/de-de.go (hunk 2 above)
Comment (chappjc) [reply client/webserver/locales/de-de.go:248]: @karamble Would you provide this one quick?  Thanks.

Comment (chappjc) [reply client/webserver/locales/de-de.go:248]: I'm guessing just "synchronisieren" but I really don't know if that's appropriate in this context.

Comment (karamble) [reply client/webserver/locales/de-de.go:248]: correct

**Pull request merged**